src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -88,7 +88,7 @@ public function findForCustomerStatistics(CustomerInterface $customer): array
     {
         return $this->createByCustomerIdQueryBuilder($customer->getId())
             ->andWhere('o.state NOT IN (:states)')
-            ->setParameter('states', [OrderInterface::STATE_CART, OrderInterface::STATE_CANCELLED])
+            ->setParameter('states', [OrderInterface::STATE_NEW, OrderInterface::STATE_CANCELLED])
```

Comment:
Shouldn't the cart s be excluded also?

Reply:
And it is in `createByCustomerIdQueryBuilder($customerId)`

Reply:
Shouldn't we get only `OrderInterface::STATE_FULFILLED` carts?

Reply:
And we get only `OrderInterface::STATE_FULFILLED`. I know that this line is not fully readable, but when I tried:
```
->andWhere('o.state = :state')
->setParameter('state', OrderInterface:: STATE_FULFILLED)
```
it doesn't work, because of parameter which was set in `createByCustomerIdQueryBuilder($customerId)`

Reply:
Hmm, what about not using that `createByCustomerIdQueryBuilder` at all here? It looks tricky.